actions/testmo-create-resources/action.yml

```diff
@@ -0,0 +1,38 @@
+name: create testmo resources
```

Comment:
Where do we call this action from? Can you provide an example of calling it?

Reply:
I'll be adding the following step to the `.github/workflows/nm-lm-eval.yml` file in the nm-vllm-ent repo:

```
      - name: create testmo resources
        id: create_testmo_resources
        uses: ./github/actions/nm-testmo-create-resources/
        if: (success() || failure()) && (inputs.testmo_run_id != '')
        with:
          resources_file: testmo_lmeval_resources.json
          python: ${{ inputs.python }}
          runner: ${{ inputs.test_label }}
          lm_eval_configuration: ${{ inputs.lm_eval_configuration }}
```

Reply:
ok got it, again let's not pass in each field as argument but use the test_config and do the parsing in the nm-testmo-create-resources action. Also maybe we should rename the action to nm-testmo-create-thread-resources as it's not for the testmo run but for each thread.

Reply:
I'll work on a way to pass in something other than hard coded fields.  That makes this action even more generic. 
 Any `testmo automation:run:*` command can accept a resources.json file, so it makes sense to leave the name as is.